django/db/models/base.py

```diff
@@ -281,7 +281,7 @@ def __new__(cls, name, bases, attrs, **kwargs):
                             )
                         )
                 else:
-                    new_class.add_to_class(field.name, copy.deepcopy(field))
+                    new_class.add_to_class(field.name, field.copy_for_child())
```

Comment:
OK, so this is the worry. We're adding `copy_for_child()` API here. 

Are the `contenttypes` fields the only possible value for `field` at this point? (Probably not, yes?)